x230/README.md

```diff
@@ -18,32 +18,29 @@ Get it from our [release page](https://github.com/merge/coreboot-x230/releases)
 * [How to rebuild](#how-to-reproduce-the-release-images)
 
 ## TL;DR
-1. run `sudo ./x230_before_first_install.sh` on your current X230 Linux system
-2. Power down, remove the battery. Remove the keyboard and palmrest. Connect
-a hardware flasher to an external PC (or a Raspberry Pi with a SPI 8-pin chip clip
-can directly be used), and run
-`sudo ./external_install_bottom.sh` on the lower chip
-and `sudo ./external_install_top.sh` on the top chip of the two.
-3. For updating later, run `./x230_skulls.sh`. No need to disassemble.
+1. If your Thinkpad is already running linux: run `sudo ./x230_before_first_install.sh` on it
+2. Power down, remove the battery. Remove the keyboard and palmrest.
+3. Connect a hardware flasher to an external PC (or a Raspberry Pi with a SPI 8-pin chip clip
+can directly be used)
+4. Run `sudo ./external_install_bottom.sh` on the lower chip
+5. Run `sudo ./external_install_top.sh` on the top chip of the two
+6. Optionally: For updating later, run `./x230_skulls.sh`. No need to disassemble.
 
 And always use the latest [released](https://github.com/merge/coreboot-x230/releases)
-package. This will be tested. The git master
-branch is _not_ meant to be stable. Use it for testing only.
+package. This will be tested. The git master branch is _not_ meant to be stable. Use it for testing only.
```

Comment:
this is all random refactoring. and _if_ you want that should be a seperate commit

Reply:
True, just happened to fix it in passing. Not important.